### DISCUSSION TITLE
splitChunks maxSize/minSize/maxRequest priority

### DISCUSSION THREAD

**Discussion by hai-x** [webpack/webpack#17970] [Q&A] [answered]
As stated on the website
[maxSize takes higher priority than maxInitialRequest/maxAsyncRequests. Actual priority is maxInitialRequest/maxAsyncRequests < maxSize < minSize.](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunksmaxsize)

After reading the source code, I found that the actual priority should be minSize>maxRequest>maxSize. 
- minSize check https://github.com/webpack/webpack/blob/0781eac69cf26af59b51513b3da9569f653ec64d/lib/optimize/SplitChunksPlugin.js#L1309
- maxRequests check 
https://github.com/webpack/webpack/blob/0781eac69cf26af59b51513b3da9569f653ec64d/lib/optimize/SplitChunksPlugin.js#L1419

- maxSize check
https://github.com/webpack/webpack/blob/0781eac69cf26af59b51513b3da9569f653ec64d/lib/optimize/SplitChunksPlugin.js#L1642

Not sure if I understood it wrong or the tips on the official website is wrong.

**Comment by alexander-akait** [comment, marked as answer]
@haijie-x Sorry for delay, feel free to send a fix to our docs

**Comment by hai-x** [comment]
@alexander-akait thanks for your reply. related PR https://github.com/webpack/webpack.js.org/pull/7123
